FIAT/macro.py

```diff
@@ -485,6 +485,61 @@ def __init__(self, ref_el, degree, order=1, vorder=None, shape=(), **kwargs):
         super().__init__(ref_el, degree, degree, expansion_set, coeffs)
 
 
+def hdiv_conforming(U, order=0):
```

Comment:
doc string to say what this does?  Tiles a general Piola-mapped H(div) element?

Reply:
Done. Somethimes we cannot just tile a CiarletElement, in particular for the H(div, S) case. This function computes the nullspace of the normal jump constraints for vector- or tensor- valued `PolynomialSet`.